CMakeLists.txt

```diff
@@ -41,14 +50,47 @@ include(BuildSettings)
 include(Packages)
 include(ClangFormat)
 
-################################################################################
-#   tomopy source
-################################################################################
+# ##############################################################################
+# tomopy source
+# ##############################################################################
 
 add_subdirectory(source)
 
-################################################################################
-#   Information
-################################################################################
+get_property(libtomo_TARGETS GLOBAL PROPERTY libtomo_TARGETS)
+
+# ##############################################################################
+# Information
+# ##############################################################################
 
 print_features()
+
+install(
+  EXPORT libtomoTargets
+  FILE libtomoTargets.cmake
+  NAMESPACE libtomo::
```

Comment:
Basically this line here means that, in the install tree, the targets in this export set will be prefixed with `libtomo::` when you find the package. E.g. you created a `tomo-prep` target in the build tree so, when it gets imported from the install tree, it will be `libtomo::tomo-prep`

Reply:
Thanks! I was making a mistake in two places:
1. need to use "libtomo::tomo-prep" instead of "tomo-prep"
3. include statements was "libtomo/prep.h" instead of "prep.h"